apps/gallery/js/startup.js

```diff
@@ -273,11 +273,17 @@
       window.performance.mark('contentInteractive');
 
       // Remove the attribute to trigger gaia header font-fit logic, defer it
-      // til now to postpone the overhead.
-      var headers = document.querySelectorAll('gaia-header');
-      for (var i = 0; i < headers.length; ++i) {
-        headers[i].removeAttribute('no-font-fit');
-      }
+      // til now to postpone the overhead. Add setTimeout to lazyload
+      // gaia-header.js aynchronously.
+      setTimeout(function() {
```

Comment:
I don't really understand that bug. LazyLoader.load() is async, though I guess it could be even more async. Maybe at this point we need to return to the event loop as soon as possible, to get stuff on the screen. If so, then using setTimeout is the right thing to do.